### PR TITLE
Gibs rework 2: Electric Boogaloo

### PR DESCRIPTION
**Pull request by yoyobatty**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl:
add: Gibs monster has a very small chance of reproducing when eating.
fix: Fixed Gibs monster getting stuck eating one mob.
balance: Gibs monster moves and attacks faster.
fix: Gibs nobody longer has the horrorform login message
tweak: Gibs will appear frustrating if it fails to get it's meal at low health.
balance: Gibs are a little bit more smarter now.
/:cl:

## About The Pull Request
I tested this new mob out on the live server and discovered a lot of issues with it. Getting stuck eating the same dead mob forever, moving way too slowly, not appearing very intelligent, attacking rather slow, etc...
This will give them a significant buff, although a few variables will be subject to tweaking as testing continues.

## Why It's Good For The Game
I think this mob should have enough bite to live up to it's bark. I am curious to see how the playerbase will react to it.

### CODE REVIEW

**Comment by Majick1** [comment]
+1 we'll do it live

**Comment by bigbluebeaker** [comment]
+1 interested in seeing how this plays out

**Comment by Manly-Dwarf** [comment]
farty party all the time

**Comment by cris5311** [comment]
+1 
cant see how to not +1 this

File: hippiestation/code/modules/mob/living/simple_animal/hostile/sentientgibs.dm [hunk 1]
```diff
@@ -34,47 +44,49 @@
...
-				to_chat(lunch, "<span class='userdanger'>[src] takes a huge bite out of you!</span>")
-				var/obj/effect/decal/cleanable/blood/gibs/G = new(get_turf(lunch))
-				step(G, pick(GLOB.alldirs)) //Make some gibs spray out for dramatic effect
-				playsound(lunch, 'sound/effects/splat.ogg', 50, 1)
-				playsound(lunch, 'hippiestation/sound/misc/tear.ogg', 50, 1)
-				lunch.emote("scream")
-				adjustBruteLoss(-50)
-	. = ..()
+			if(!EatLunch(lunch) && (health < maxHealth * 0.25) && prob(33))//Don't want to see this message all the time.
+				visible_message("<span class='warning'>[lunch] appears frustrated!</span>", \
```
Comment (ghost) [inline hippiestation/code/modules/mob/living/simple_animal/hostile/sentientgibs.dm]: Firstly, this message is kind of dry. Secondly, it should be [src], not [lunch] as the subject here.
You should probably throw in a jitter effect here as well as something like "[src] vibrates in exasperation!"

**Review by ghost — CHANGES_REQUESTED** [review]
minor nitpick but otherwise it looks okay

File: hippiestation/code/modules/mob/living/simple_animal/hostile/sentientgibs.dm (hunk 1 above)
Comment (yoyobatty) [reply hippiestation/code/modules/mob/living/simple_animal/hostile/sentientgibs.dm]: Done

**Review by ghost — APPROVED** [review]

**Pull request merged**